minio/console
# Show object versions on UI only when bucket has versioning enabled

fixes https://github.com/minio/console/issues/594
If bucket doesn't have versioning enabled, objects shouldn't show component that lists versions.

<img width="576" alt="Screen Shot 2021-03-09 at 2 43 02 PM" src="https://user-images.githubusercontent.com/11819101/110545246-72b77a00-80f2-11eb-8393-4da626cd482a.png">
<img width="636" alt="Screen Shot 2021-03-09 at 2 45 35 PM" src="https://user-images.githubusercontent.com/11819101/110545257-7519d400-80f2-11eb-86c8-cb249a1c25ea.png">

To test:
Create a bucket without versioning, add an object and on the object details the list shouldn't appear.
Create a bucket with versioning, add an object and on the object details the list should appear.
